proto/Networking/Requests/Messages/ChangeTeamMessage.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+package POGOProtos.Networking.Requests.Messages;
+
+import "POGOProtos/Inventory/Item/ItemId.proto";
+
+message ChangeTeamMessage {
+	.POGOProtos.Inventory.Item.ItemId item = 1;
+	.POGOProtos.Inventory.Item.ItemId team = 2;
```

Comment:
This is a Team, not an Item

Reply:
ok but this need special item also is item team maybe item is ok

Reply:
```proto
import "POGOProtos/Enums/TeamColor.proto";
import "POGOProtos/Inventory/Item/ItemId.proto";

message ChangeTeamMessage {
	.POGOProtos.Inventory.Item.ItemId item = 1;
	.POGOProtos.Enums.TeamColor team = 2;
}
```